api/post_test.go

```diff
@@ -1153,20 +1153,31 @@ func TestEmailMention(t *testing.T) {
 		if !strings.ContainsAny(resultsMailbox[len(resultsMailbox)-1].To[0], th.BasicUser2.Email) {
 			t.Fatal("Wrong To recipient")
 		} else {
-			for i := 0; i < 5; i++ {
-				if resultsEmail, err := utils.GetMessageFromMailbox(th.BasicUser2.Email, resultsMailbox[len(resultsMailbox)-1].ID); err == nil {
-					if strings.Contains(resultsEmail.Body.Text, post1.Message) {
-						break
-					} else if i == 4 {
-						t.Log(resultsEmail.Body.Text)
-						t.Fatal("Received wrong Message")
+			for i := 0; i < 30; i++ {
```

Comment:
Since other tests are generating emails while this test is running, I had to modify this to ignore all emails sent to addresses with different plus suffixes.